README.md

```diff
@@ -124,6 +124,42 @@ a working system.
    $ export FISSILE_STEMCELL=splatform/fissile-stemcell-opensuse:$FISSILE_STEMCELL_VERSION
    ```
 
+3. Environment variables to configure `vagrant up` (optional)
+    - `VAGRANT_VBOX_BRIDGE`: Set this to the name of an interface to enable bridged networking when
+      using the Virtualbox provider. Turning on bridged networking will allow your vagrant box to receive
+      an IP accessible anywhere on the network. While Virtualbox is able to bridge over an interface
+      without any special networking configuration (and may even do this on OSX), bridged networking may
+      not be supported when the provided interface is a wireless interface.
```

Comment:
Maybe link to the docs here? (maybe [this link](https://www.virtualbox.org/manual/ch06.html#network_bridged)?)